src/EPR.SubsidiaryBulkUpload.Function.UnitTests/BulkUploadFunctionTests.cs

```diff
@@ -71,11 +71,23 @@ public void TestInitialize()
     [TestMethod]
     public async Task BulkUploadFunction_Calls_CsvService()
     {
+        var downloadStreamingDetails = BlobsModelBuilder.CreateBlobDownloadDetails(
+            CsvContent.Length,
+            new Dictionary<string, string> { { "UserId", Guid.NewGuid().ToString() }, { "OrganisationId", "TestOrgId" } });
+
+        using var stream = new MemoryStream(Encoding.UTF8.GetBytes(CsvContent));
+        var downloadStreamingResult = BlobsModelFactory.BlobDownloadStreamingResult(stream, downloadStreamingDetails);
+
+        var response = Response.FromValue(downloadStreamingResult, new Mock<Response>().Object);
+        _blobClientMock.Setup(client => client.DownloadStreamingAsync(It.IsAny<BlobDownloadOptions>(), It.IsAny<CancellationToken>()))
+            .ReturnsAsync(response);
+
         // Act
         await _systemUnderTest.Run(_blobClientMock.Object);
 
         // Assert
         _csvProcessorMock.Verify(x => x.ProcessStreamWithMapping<CompaniesHouseCompany, CompaniesHouseCompanyMap>(It.IsAny<Stream>(), It.IsAny<CsvConfiguration>()), Times.Once);
+        _loggerMock.VerifyLog(x => x.LogInformation("Blob trigger processed {Count} records from csv blob {Name}", CsvRowCount, CsvBlobName), Times.Once);
```

Comment:
Why are we always verifying logger messages?
It feels rather alien to me.

Reply:
Just followed the other logger unit tests to make sure verify correct logging message in place.